zproject/dev_settings.py

```diff
@@ -85,3 +85,6 @@
 
 # Make sendfile use django to serve files in development
 SENDFILE_BACKEND = 'sendfile.backends.development'
+
+# Set this True to send all hotspots in development
+ALWAYS_SEND_ALL_HOTSPOTS = False  # type: bool
```

Comment:
You also need to add a default for this in `DEFAULT_SETTINGS` in /etc/zulip/settings.py; otherwise the hotspots code will crash in production.  See zulip.readthedocs.io/en/latest/subsystems/settings.html